__tests__/app.test.js

```diff
@@ -57,6 +57,94 @@ describe("GET /api/articles", () => {
         });
       });
   });
+  test("200: responds with an articles array of article objects which sorted by any valid column", () => {
+    return request(app)
+      .get("/api/articles?sort_by=title&order=asc")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toBeInstanceOf(Array);
+        expect(body.articles).not.toHaveLength(0);
+        expect(body.articles).toBeSortedBy("title", {
+          ascending: true,
+        });
+        body.articles.forEach((article) => {
+          expect(article).toEqual(
+            expect.objectContaining({
+              article_id: expect.any(Number),
+              title: expect.any(String),
+              topic: expect.any(String),
+              author: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              comment_count: expect.any(Number),
```

Comment:
As you have previously used a forEach to assert the structure of the articles in another test, there is no need to do so for these query tests. Asserting that the articles have been sorted correctly will do nicely and will DRY up your test suite